### ISSUE TITLE
OC-05 Display Board

### ISSUE THREAD

**Issue by joseph-allen** [joezcool02/OurChess#5]
Display the board and its contents

**Comment by 3DIndian** [comment]
First we need the images(chess pieces) to display! Which images do we use??

**Comment by joseph-allen** [comment]
for now we can just use characters
P - Pawn
Q - Queen 
etc.

we need to start somewhere else first though, we need a board before we try to display the board :)

**Comment by 3DIndian** [comment]
Ok!
so we'll create an 8x8 integer array which stores the values of the chess board, where we can map Pawn to 1, Queen to 2 and so on

**Comment by joseph-allen** [comment]
yes, that would be for the Create Board Issue perhaps, however now we have just merged one PR which solves quite a lot of issues so have a second look :)
